### DISCUSSION TITLE
how do i insert javascript in my html

### DISCUSSION THREAD

**Discussion by BobbyJeff-Buffguy** [community/community#38193] [Other Feature Feedback, Questions, & Ideas]
how do i insert prosesing JS into html

**Comment by EliasAfara** [comment]
Check out [this](https://www.digitalocean.com/community/tutorials/how-to-add-javascript-to-html) article, it explains everything in details. 
Also I recommend to check out [w3schools JS](https://www.w3schools.com/js/js_whereto.asp).

**Reply by EliasAfara** [reply]
@BobbyJeff-Buffguy mark it as an answer to close the discussion.

**Comment by david-engelmann** [comment]
This is probably not the best location for this inquiry

**Comment by DSDmark** [comment]
Hi @BobbyJeff-Buffguy,

can you're more specific of what you want to do? If you want to use JavaScript in HTML then that is perfect article about that which was @EliasAfara wrote or if you want to write JavaScript as is it then you can use ```<code>``` tag. here is more about that:- [**Code Tag**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code)

**Comment by chandran-jr** [comment]
Use a script tag bro.
<script> Your JavaScript Here </script>
Preferably put the script tags in the bottom of the body tag because if your JS is heavy, the website can take a lot of time to load up which eventually means it is not optimized. Hope it helps. Mark my answer as correct if you can 😊

**Comment by GDSimpson3** [comment]
you can either use a script tag like so:

<script> 
function functionName(){
alert('this is an alert, who would have guessed')
}
</script>

or import it:

<script src="myJSfile.js"></script>

**Reply by khaledalam173** [reply]
Thanks
